src/main/java/org/swisspush/reststorage/LoggingFileReadStream.java

```diff
@@ -78,11 +75,17 @@ public ReadStream resume() {
         return this;
     }
 
+    @Override
```

Comment:
not needed?

Reply:
yes, it is needed, resume is newly added to Vertx interface, so I need call delegate.resume